### PR TITLE
Restored stack analyses integration test

### PR DESCRIPTION
**Pull request by samuzzal-choudhury**

### CODE REVIEW

**Comment by miteshvp** [comment]
Lets test it on Stage

File: integration-tests/features/steps/stack_analysis.py [hunk 1]
```diff
@@ -110,9 +110,12 @@ def stack_analysis_endpoint(context, version):
...
     # Two available endpoints for stack analysis are /stack-analyses and /analyse
     # /analyse endpoint was developed to meet the performance norms at production
-    endpoint = {"1": "/api/v1/stack-analyses-v1",
-                "2": "/api/v1/analyse",
-                "3": "/api/v1/stack-analyses"}.get(version)
+    endpoint_arr = ["/api/v1/stack-analyses-v1",
+                    "/api/v1/analyse",
+                    "/api/v1/stack-analyses/"]
+    index = int(version) - 1
+    endpoint = endpoint_arr[index]
```
Comment (tisnik) [inline integration-tests/features/steps/stack_analysis.py]: Please note that this statement can fail now. Before it was checked by next statement `if endpoint is None:`

Comment (tisnik) [reply integration-tests/features/steps/stack_analysis.py]: also, we can force Behave to store version as an int.

File: integration-tests/features/steps/stack_analysis.py [hunk 2]
```diff
@@ -149,7 +152,7 @@ def python_manifest_stack_analysis(context, manifest, version="2", token="withou
 @when("I send Maven package manifest {manifest} to stack analysis version {version}")
```
Comment (tisnik) [inline integration-tests/features/steps/stack_analysis.py]: you can add `:d` here

**Pull request merged**